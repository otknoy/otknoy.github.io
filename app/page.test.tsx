import { test, describe, expect, vi } from 'vitest'
import { render } from '@testing-library/react'
import * as React from 'react'

import Home from './page'

vi.useFakeTimers()
vi.setSystemTime(new Date('2025-01-01'))

describe('render <Home />', () => {
  test('snapshot', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
})
