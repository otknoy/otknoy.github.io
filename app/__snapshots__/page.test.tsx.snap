// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <Home /> snapshot 1`] = `
<div>
  <div>
    <header
      style={
        {
          "display": "flex",
          "justifyContent": "space-between",
        }
      }
    >
      <h1
        style={
          {
            "fontSize": "24px",
          }
        }
      >
        Naoya Otsuka
      </h1>
      <ul
        style={
          {
            "listStyle": "none",
          }
        }
      >
        <li
          style={
            {
              "display": "inline-block",
              "padding": "4px",
            }
          }
        >
          <a
            href="https://github.com/otknoy"
          >
            <span
              style={
                {
                  "fontSize": "16px",
                }
              }
            >
              GitHub
            </span>
          </a>
        </li>
        <li
          style={
            {
              "display": "inline-block",
              "padding": "4px",
            }
          }
        >
          <a
            href="https://twitter.com/otknoy"
          >
            <span
              style={
                {
                  "fontSize": "16px",
                }
              }
            >
              X
            </span>
          </a>
        </li>
        <li
          style={
            {
              "display": "inline-block",
              "padding": "4px",
            }
          }
        >
          <a
            href="https://bsky.app/profile/otknoy.bsky.social"
          >
            <span
              style={
                {
                  "fontSize": "16px",
                }
              }
            >
              bsky
            </span>
          </a>
        </li>
        <li
          style={
            {
              "display": "inline-block",
              "padding": "4px",
            }
          }
        >
          <a
            href="https://otknoy.hatenablog.com"
          >
            <span
              style={
                {
                  "fontSize": "16px",
                }
              }
            >
              blog
            </span>
          </a>
        </li>
      </ul>
    </header>
    <section>
      <h1
        style={
          {
            "fontSize": "22px",
          }
        }
      >
        Profile
      </h1>
      <div>
        <p>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            web engineer, architect
          </span>
        </p>
      </div>
    </section>
    <section>
      <h1
        style={
          {
            "fontSize": "22px",
          }
        }
      >
        Skills
      </h1>
      <ul>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Java, Kotlin, SpringBoot
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Go, Python
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Docker, Kubernetes, Microservices
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Solr
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Information Retrieval
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Human Computer Interaction
          </span>
        </li>
        <li>
          <span
            style={
              {
                "fontSize": "16px",
              }
            }
          >
            Exploratory Search
          </span>
        </li>
      </ul>
    </section>
    <footer
      style={
        {
          "textAlign": "center",
        }
      }
    >
      <h1
        style={
          {
            "margin": "0px",
            "padding": "4px",
          }
        }
      >
        <span
          style={
            {
              "fontSize": "16px",
            }
          }
        >
          (C) 1988-2025 Naoya Otsuka
        </span>
      </h1>
    </footer>
    <style>
      
        div {
          max-width: 410px;
          margin: 0 auto;
          padding: 0 8px;
        }
      
    </style>
  </div>
</div>
`;
